src/Nethermind/Nethermind.Core/BlockchainIds.cs

```diff
@@ -27,7 +27,7 @@ namespace Nethermind.Core
     /// 7762959: Musicoin, the music blockchain
     /// 61717561: Aquachain, ASIC resistant chain
     /// </summary>
-    public static class ChainId
+    public static class BlockchainIds
```

Comment:
Why rename? I thought it make sense before.

Reply:
Because it's both networkid and chainid, when we use it in a context when we need networkid it looks lie: networkId = ChainId.Mainnet. and in opposite. So I named it more generically.

Reply:
What about chain with different chainid and network id?

Reply:
Public chains seem to have both values always the same. The difference appears for devnets/testnets, and networkid/chainid can be configured via specs for them.

I would postpone resolution for such a case to the time when we will want to add a well-known network to this list with networkid!=chainid. I see no not-ugly solution for it for now:
- if I duplicate the class so we will have almost the same ChainIds and NetwortkIds, it becomes an additional cause for bugs
- if I make a class that inherits ( class ChainIds{}; class NetworkIds:ChainIds{} ), there will be unbeatable warnings from resharper
- no other idea

Reply:
Ok, fair enough.